lib/core/src/display/rendering/dom_container.rs

```diff
@@ -27,15 +36,19 @@ pub trait ResizeCallbackFn = where Self: Fn(&Vector2<f32>) + 'static;
 #[derive(Derivative)]
 #[derivative(Debug)]
 pub struct DOMContainerData {
+    position   : Vector2<f32>,
     dimensions : Vector2<f32>,
     #[derivative(Debug="ignore")]
-    resize_callbacks : Vec<ResizeCallback>
+    resize_callbacks       : Vec<ResizeCallback>,
```

Comment:
I dont like keeping callbacks as explicit vec of dyn fns. We've got a callback manager somewhere. If you need to add / remove callbacks and keep list of them, we should use everywhere a callback manager for that. If the one we've got is not suiting your needs, lets talk about it!

Reply:
If you are talking about `controls/callback`, is there any way to define callbacks with parameters with it?